FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/workmanager/HandlePasswordProtectedMsgWorker.kt

```diff
@@ -135,11 +137,18 @@ class HandlePasswordProtectedMsgWorker(context: Context, params: WorkerParameter
 
             //start of creating and uploading a password-protected msg to FES
             val fromAddress = plainMimeMsgWithAttachments.getFromAddress()
-            val domain = EmailUtil.getDomain(fromAddress.address)
+            val baseFesUrlPath = if (account.useFES) {
+              "fes." + EmailUtil.getDomain(fromAddress.address)
+              //for example fes.flowcrypt.com
+            } else {
+              val url = URL(BuildConfig.API_URL)
+              (url.host + url.path).replace("/$".toRegex(), "")
+              //for example flowcrypt.com/shared-tenant-fes
+            }
```

Comment:
Regarding these 4 lines, seems to me these are likely duplicated elsewhere too? Since this is just for `prepareAndUploadPasswordProtectedMsgsToFES` and there are surely other situations where this decision has to be made? (eg getting client configuration)